test/c++/backend_test.cpp

```diff
@@ -60,6 +60,17 @@ TEST_F(BackendTest, SimpleVarDecl) {
     EXPECT_EQ (0, basicTest(is));
 }
 
+
```

Comment:
(1) Can we add a test case that declares a global const variable that is of uint type, and later increment it in the main function? Similar to the code in Triangle counting?
(2) Add another test case that declares a const variable with no initial value, and later assigning it a value. 
(3) Add yet another test case that declares a local variable (in the main function) of type uint / uint64_t? The two test cases are all of global (const) variables outside of a function declaration.

Reply:
got it